app/src/main/java/org/mozilla/focus/notification/BrowsingNotificationService.java

```diff
@@ -70,6 +70,7 @@ public static void stop(Context context) {
     public int onStartCommand(Intent intent, int flags, int startId) {
         switch (intent.getAction()) {
             case ACTION_START:
+                foreground = true;
```

Comment:
Add a comment explaining why this is necessary (because we stop the Service with ACTION_STOP so it loses the Activities' foreground state and we can assume when ACTION_START is called, the Activity is in the foreground).